src/routers/RouteRouter.js

```diff
@@ -63,20 +64,62 @@ class RouteRouter extends AppDevRouter<Array<Object>> {
         const errors = [];
 
         try {
-            busRoute = await axios.get(`http://${process.env.GHOPPER_BUS || 'ERROR'}:8988/route`, {
-                params: parameters,
-                paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' }),
-            });
+            const options = {
+                method: 'GET',
+                url: `http://${process.env.GHOPPER_BUS || 'ERROR'}:8988/route`,
+                qs:
+                    {
+                        vehicle: 'pt',
+                        weighting: 'short_fastest',
+                        elevation: false,
+                        point: [start, end],
+                        points_encoded: false,
+                    },
+            };
+
+            busRoute = JSON.parse(await new Promise((resolve, reject) => {
+                request(options, (error, response, body) => {
+                    if (error) reject(error);
+                    console.log(response);
+                    resolve(body);
+                });
+            }).then(value => value).catch((error) => {
+                errors.push(ErrorUtils.log(routeErr, parameters, `Routing failed: ${process.env.GHOPPER_BUS || 'undefined graphhopper bus env'}`));
+                return null;
+            }));
+
+            console.log('busRoute req', busRoute);
+        
         } catch (routeErr) {
             errors.push(ErrorUtils.log(routeErr, parameters, `Routing failed: ${process.env.GHOPPER_BUS || 'undefined graphhopper bus env'}`));
             busRoute = null;
         }
 
         try {
-            walkingRoute = await axios.get(`http://${process.env.GHOPPER_WALKING || 'ERROR'}:8987/route`, {
-                params: walkingParameters,
-                paramsSerializer: (params: any) => qs.stringify(params, { arrayFormat: 'repeat' }),
-            });
+            const options = {
+                method: 'GET',
+                url: `http://${process.env.GHOPPER_WALKING || 'ERROR'}:8987/route`,
+                qs:
+                    {
+                        vehicle: 'foot',
+                        point: [start, end],
+                        points_encoded: false,
+                    },
+            };
+
+            walkingRoute = JSON.parse(await new Promise((resolve, reject) => {
+                request(options, (error, response, body) => {
```

Comment:
this code snippet with `if (error) reject(error) ... resolve(body)` shows up multiple times, there's a better way to do this for sure -> i agree with @AAAstorga that this can be refactored